### DISCUSSION TITLE
get started for this project

### DISCUSSION THREAD

**Discussion by karleeov** [karleeov/chatgpt-aoai#4] [Q&A] [answered]
if any questions to start we can discuss here

**Comment by karleeov** [comment, marked as answer]
quick reference need to use azure auth login to install az auth login can refer this link
https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli-interactively

**Comment by mikeng9944** [comment]
Why choose clear as the login?

**Reply by karleeov** [reply]
u mean azure? is for the azure cloud to deploy your project and create your azure resources

**Comment by nathanhazedawn** [comment]
What alternative authentication methods are recommended for production-level applications using the Azure CLI?

**Comment by nathanhazedawn** [comment]
How can you initiate the device code flow for authentication in the Azure CLI?
